src/main/webapp/app/entities/course/course.component.html

```diff
@@ -41,35 +41,9 @@ <h4>
                 <td>{{course.onlineCourse}}</td>
                 <td class="text-right">
                     <div class="btn-group flex-btn-group-container">
-                        <div class="d-inline-block" ngbDropdown>
-                            <button class="btn btn-primary btn-sm mr-1" ngbDropdownToggle>
-                                <fa-icon [icon]="'list-alt'"></fa-icon>
-                                <span class="hidden-xs hidden-sm" jhiTranslate="arTeMiSApp.course.exercises"></span>
-                                <span class="caret"></span>
-                            </button>
-                            <div ngbDropdownMenu class="dropdown-menu">
-                                <button class="dropdown-item"
-                                        routerLink="{{course.id}}/programming-exercise"
-                                        jhiTranslate="arTeMiSApp.course.programmingExercises"
-                                ></button>
-                                <button class="dropdown-item"
-                                        routerLink="{{course.id}}/quiz-exercise"
-                                        jhiTranslate="arTeMiSApp.course.quizExercises"
-                                ></button>
-                                <button class="dropdown-item"
-                                        routerLink="{{course.id}}/modeling-exercise"
-                                        jhiTranslate="arTeMiSApp.course.modelingExercises"
-                                ></button>
-<!--                                <button class="dropdown-item"
-                                        routerLink="{{course.id}}/text-exercise"
-                                        jhiTranslate="arTeMiSApp.course.textExercises"
-                                ></button>
-                                <button class="dropdown-item"
-                                        routerLink="{{course.id}}/file-upload-exercise"
-                                        jhiTranslate="arTeMiSApp.course.fileUploadExercises"
-                                ></button>-->
-                            </div>
-                        </div>
+                        <button type="submit" [routerLink]="['/course', course.id ]" class="btn btn-dark btn-sm mr-1">
+                            <span class="d-none d-md-inline">Exercises</span>
```

Comment:
Please include translation and use the same design as the old Exercise button, i.e. with the same icon and the sam color:
<img width="111" alt="image" src="https://user-images.githubusercontent.com/744067/49731033-58c03000-fc7a-11e8-85b8-a65d0d45d930.png">